libnmstate/state.py

```diff
@@ -45,35 +45,78 @@
 from libnmstate.schema import InterfaceState
 from libnmstate.schema import InterfaceType
 from libnmstate.schema import Route
+from libnmstate.schema import RouteRule
 
 
 NON_UP_STATES = (InterfaceState.DOWN, InterfaceState.ABSENT)
 
 
 @total_ordering
-class RouteEntry(object):
+class _StateObject(object):
```

Comment:
How about `StateEntry` ?
BTW, with py3 you do not need to inherit from `Object` any-more.

In general, it would have been cleaner to post this change as a separate commit and show everything continues to work as before and then in the next commit use it.

Reply:
It gets harder to follow what changed in this conversion to the common based class.
It will help if you create for this its own commit.

Reply:
I have created PR #611 for the `RouteEntry` and `RouteRuleEntry`.